### PR TITLE
House keeping

### PR DESCRIPTION
**Pull request by pirasanthan-jesugeevegan**
- Added Husky to have a commit hook to lint/formate code 
- Changes folder/file structure 
- Change filename convention 
- Added Prettier and ESlint

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            # Open Metro Bundler in Background
+            - name: Open Metro Bundler in Background
+              run: |
+                  node_modules/react-native/scripts/launchPackager.command &
+            # Build Android App and Run Tests
+            - name: Build Android App and Run Tests
+              env:
+                  E2E_TEST_PLATFORM: 'android'
+                  E2E_TEST_DEFAULTS: 'true'
+                  E2E_TEST_TAGS: 'E2E'
```
Comment (darrenb-ta) [inline main.yml:74]: So these are the tags you mentioned yesterday.

Comment (pirasanthan-jesugeevegan) [reply main.yml:74]: No this is something Im trying for CI pipeline

**Review by darrenb-ta — APPROVED** [review]
Ran locally, no issues. Read through to understand changes for new file layout / naming.

**Review by Pelumi-Festicket — APPROVED** [review]
reviewed, everythings working fine when ran locally 👍🏾

**Pull request merged**